reconcile/saas_file_owners.py

```diff
@@ -195,11 +195,13 @@ def valid_diff(current_state, desired_state):
     for c in current_state_copy:
         c.pop('ref')
         c.pop('parameters')
+        c['saas_file_definitions'].pop('use_channel_in_image_tag')
```

Comment:
Does this need to be
```suggestion
        c['saas_file_definitions'].pop('use_channel_in_image_tag', None)
```
to avoid raising an exception for saas files where this attribute is absent?

Reply:
```
[efried@efried boilerplate]$ python3
Python 3.6.8 (default, Aug 18 2020, 08:33:21) 
[GCC 8.3.1 20191121 (Red Hat 8.3.1-5)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> d = dict(one=2, three=4)
>>> d.pop('five')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'five'
>>> d.pop('five', None)
>>> 
```

Reply:
it's always there: https://github.com/app-sre/qontract-reconcile/blob/b16e1a014f7942e7cd57f928313be71eac724f25/reconcile/saas_file_owners.py#L65

Reply:
Okay.